packaging/flavorgen/flavors/flavors.go

```diff
@@ -88,6 +88,39 @@ func MultiNodeTemplateWithKubeVIP() []runtime.Object {
 	return append(MultiNodeTemplate, crsResources...)
 }
 
+// MultiNodeTemplateWithWindows generates templates for a hybrid cluster with Linux and Windows worker nodes
+func MultiNodeTemplateWithWindows() []runtime.Object {
+	vsphereCluster := newVSphereCluster(nil)
+	linuxMachineTemplate := newVSphereLinuxMachineTemplate()
+	controlPlane := newKubeadmControlplane(445, linuxMachineTemplate, newKubeVIPFiles())
```

Comment:
not important but curious what this # is maybe a comment for it?

Reply:
ah that's another one of those placeholder replicas :) I'm not sure why the high numbers were used, just grabbed it from the Linux templates

Reply:
imo 123 would be less likely to cause a red -herring :) 445 almost seems to imply a permissions setting or somethin

Reply:
it's file mode, so perms.